dash-pipeline/SAI/templates/saifixedapis.cpp.j2

```diff
@@ -55,8 +56,8 @@ sai_status_t sai_create_switch_dummy(
 #define DASH_BMV2_DEFAULT_1Q_BRIDGE_ID MAKE_OID(SAI_OBJECT_TYPE_BRIDGE,1)
 
 sai_object_id_t port_list[DASH_BMV2_NUM_PORTS] = {
-    MAKE_OID(SAI_OBJECT_TYPE_PORT,1),
-    MAKE_OID(SAI_OBJECT_TYPE_PORT,2)
+    (sai_object_id_t)0,
```

Comment:
is this creating now the proper port object of SAI_OBJECT_TYPE_PORT ? or this is still work in progress

Reply:
Work in progress

Reply:
Please add a #TODO comment if needed. I'm not sure there's anything "wrong" with this since for bmv2 we can assume just two dataplane ports, but these are not traditional "OIDs."